### ISSUE TITLE
[Windows] [1.7] possible deadlock of other applications because of Windows overlay icons

### ISSUE THREAD

**Issue by pchnl** [owncloud/client#2515]
Noticed something weird with the v1.7 of desktop client, installed on a Windows 7 x32.
Directly after installing, I found that both MS Word and Excel 2013 started to freeze.
You run - for example - Excel, everything works fine, leave it and after 5 ~ 10 minutes it is frozen. It is 100% reproducible. It concerns an AppHangB1 crash and message in Windows log is "Crossthread-Deadlock".
I uninstalled the OwnCloud sync client and the freezing continued. I than learned about the icon overlay and after some "jumping through hoops" I could remove the icon overlay shell extension DLLs from the system. This solved the problem, MS Office is rock solid again.

I don't know, but I suspect that because I have SVN tortoise 1.7.12 running on my system and the files opened in MSOffice are in a subversion local work-directory this has something to do with it (as if 2 programs attempting to overlay icons are causing a deadlock). The SVN local copy and Owncloud sync are 2 different folders (C:\OwnCloud and C:\Subversion) so it's a bit surprising if they would interfere.

Yet another example that the client needs more configuration options to be useful.

**Comment by plages** [comment]
I'm not sure if this is related but in my office Autodesk Revit Architecture 2014 hung on file >> open after installing 1.7.1. After de-installing the client and vcredit2013 everything works OK again.
I'm pretty sure this has to do with the vcredist2013 installation that comes with the client.
I've been fiddling with a custom client and earlier versions of vcredist but then i get an error during vcredist installation. The client then installs just fine but without the overlay icons. Revit however works as expected.

**Comment by pchnl** [comment]
I have installed OwnCloud 7.01, 7.02, 7.03 and 7.04 and each installation was painful (in other words give up and reinstall from scratch).  In addition, I have experienced plenty of issues (eg. high CPU usage in sync client, ghost files and folders, deadlock in other apps with the sync client, clumsy setup of sync client, unable to uninstall shell extensions,  connectivity issues, etc. etc.) so I gave up on OwnCloud completely. Does not seem to work on Windows 2012r2....  Which is a pity. If it worked reliable it would be an awesome product. 
Sorry :-)
